mayastor-test/e2e/install/install_test.go

```diff
@@ -165,32 +164,16 @@ func moacReadyPodCount() int {
 
 // create pools for the cluster
 func createPools(mayastorNodes []string) {
-	envPoolYamls := os.Getenv("e2e_pool_yaml_files")
 	poolDevice := os.Getenv("e2e_pool_device")
-	if len(envPoolYamls) != 0 {
-		// Apply the list of externally defined pool yaml files
-		// NO check is made on the status of pools
-		poolYamlFiles := strings.Split(envPoolYamls, ",")
-		for _, poolYaml := range poolYamlFiles {
-			fmt.Println("applying ", poolYaml)
-			bashcmd := "kubectl apply -f " + poolYaml
-			cmd := exec.Command("bash", "-c", bashcmd)
-			_, err := cmd.CombinedOutput()
-			Expect(err).ToNot(HaveOccurred())
-		}
-	} else if len(poolDevice) != 0 {
-		// Use the template file to create pools as per the devices
-		// NO check is made on the status of pools
-		for _, mayastorNode := range mayastorNodes {
-			fmt.Println("creating pool on:", mayastorNode, " using device:", poolDevice)
-			bashcmd := "NODE_NAME=" + mayastorNode + " POOL_DEVICE=" + poolDevice + " envsubst < " + "pool.yaml.template" + " | kubectl apply -f -"
-			cmd := exec.Command("bash", "-c", bashcmd)
-			cmd.Dir = getTemplateYamlDir()
-			_, err := cmd.CombinedOutput()
-			Expect(err).ToNot(HaveOccurred())
-		}
-	} else {
-		// No pools created
+	// Use the template file to create pools as per the devices
```

Comment:
Perhaps we should add a check that e2e_pool_device was defined, and fail if not.
Nothing will work if no pools are defined so kind of pointless, proceeding further.

Reply:
will fix 👍